source/commands/check_baca.py

```diff
@@ -0,0 +1,21 @@
+import requests
+
+BACA_URL = "https://baca.ii.uj.edu.pl/"
+
+
+def check_baca() -> bool:
+    try:
+        baca_status_code = requests.get(
+            BACA_URL, verify=False, timeout=10
```

Comment:
`verify=False`  not recommended

Reply:
Baca certificate doesn't work, can't verify connection.

Reply:
To resolve when baca certificate starts working.